community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -396,6 +392,22 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
             }
         }
 
+        configuration = importConfiguration( processors, defaultSettingsSuitableForTests, dbConfig, pageSize );
+        input = new CsvInput( nodeData( inputEncoding, nodesFiles ), defaultFormatNodeFileHeader(),
+                relationshipData( inputEncoding, relationshipsFiles ), defaultFormatRelationshipFileHeader(),
+                idType, csvConfiguration( args, defaultSettingsSuitableForTests ), badCollector,
```

Comment:
You're using `badCollector` here but you have already closed its output in the finally block above. If any bad lines are detected, your import will crash because it tries to write to a file which had been closed already.